pkg/utils/podman.go

```diff
@@ -5,12 +5,13 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
-	"github.com/containers/podman/v5/pkg/bindings/images"
-	"github.com/containers/podman/v5/pkg/domain/entities/types"
 	"os"
 	"os/exec"
 	"strings"
 
+	"github.com/containers/podman/v5/pkg/bindings/images"
+	"github.com/containers/podman/v5/pkg/domain/entities/types"
+
```

Comment:
remove the empty line (probably it will require reordering the import list)
this is not really a tls-related change, do you mind doing it on its own commit?

Reply:
Yep, will do. Do you mean keep the same commit?

Reply:
I meant moving this import lines on a different commit that the rest of the changes, like:
commit 1: Moving the import lines
commit 2:  All the TLSverify-related changes